charts/internal/gvisor/templates/daemonset-containerd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: containerd-gvisor
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: containerd-gvisor
+    helm.sh/chart: containerd-gvisor
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: containerd-gvisor
+  template:
+    metadata:
+      annotations:
+        checksum/configmp-containerd-gvisor: {{ include (print $.Template.BasePath "/configmap-containerd.yaml") . | sha256sum }}
+      labels:
+        app.kubernetes.io/name: containerd-gvisor
+    spec:
+      hostPID: true
+      hostIPC: true
+      nodeSelector:
+{{ toYaml .Values.config.nodeSelector | indent 8 }}
+      containers:
+      - name: container-runtime-gvisor-containerd
```

Comment:
Does resource requests/limits specification make sense for this container?

Reply:
What do you mean?

Reply:
cpu/memory requests/limits specification for this coantiner

Reply:
ping

Reply:
I'm not sure yet. WDYT?

Reply:
DemaonSet pod can now be evicted and reinstalled without problems. So I think resource limits are not required. See #3